command/add/add.go

```diff
@@ -63,15 +69,89 @@ func NewCommand(home string, docker client.CommonAPIClient, output terminal.Outp
 				dir = filepath.Clean(wd)
 			}
 
-			output.Info("Adding site…")
+			output.Info("Adding app…")
+
+			app := config.App{}
+
+			// are we excluding any dependencies?
+			if flagExcludeDependencies {
+				app.Excludes = []string{"node_modules", "vendor"}
```

Comment:
I assume people have a way to customizing the excludes... where does that get taken into account?

Reply:
We default to the "normal" excludes for php and node deps. This just adds those two folders to the app config and it can then be customized based on the specific app requirements. As an example, on a Symfony app, you can manually edit the excludes and ignore the `var` directory.